odoo/content.md

```diff
@@ -0,0 +1,83 @@
+# What is Odoo?
+
+Odoo, formerly known as OpenERP, is a suite of open-source business apps 
+written in Python and released under the AGPL license. This suite of 
+applications covers all business needs, from Website/Ecommerce down to 
+manufacturing, inventory and accounting, all seamlessly integrated. It is the 
+first time ever a software editor managed to reach such a functional coverage. 
+Odoo is the most installed business software in the world. Odoo is used by 
+2.000.000 users worldwide ranging from very small companies (1 user) to very 
+large ones (300 000 users). 
+
+> [www.odoo.com](https://www.odoo.com)
+
+%%LOGO%%
+
+# How to use this image
+
+This image requires a running PostgreSQL server. 
+
+## Start a PostgreSQL server
+
+	docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo --name db postgres
+
+## Start an Odoo instance
+
+	docker run -p 127.0.0.1:8069:8069 --name odoo --link db:db -t odoo
+
+The alias of the container running Postgres must be db for Odoo to be able
+to connect to the Postgres server.
+
+## Stop and restart an Odoo instance
+	
+	docker stop odoo
+	docker start -a odoo
+
+## Stop and restart a PostgreSQL server
+When a PostgreSQL server is restarted, the Odoo instances 
+linked to that server must be restarted as well because the server address has 
+changed and the link is thus broken. 
+
+Restarting a PostgreSQL server does not affect the created databases.
+
+## Install the latest version of Odoo
+The Odoo package is built every night to include latest functionalities and improvements.
+To install the latest version of Odoo, enter the container of your Odoo instance
+and update your version of Odoo using `apt-get`.
+
+	docker exec -it odoo bash
+	# apt-get update
+	# apt-get install odoo
+
+## Run Odoo with a custom configuration
+
+The default configuration file for the server (located at `/etc/odoo/openerp-server.conf`)
+can be overriden at startup using volumes. Suppose you have a custom configuration
+at `/path/to/config/openerp-server.conf`, then
+
+	docker run -v /path/to/config:/etc/odoo -p 127.0.0.1:8069:8069 --name odoo --link db:db -t odoo
+
+## Run multiple Odoo instances
+	
+	docker run -p 127.0.0.1:8070:8069 --name odoo2 --link db:db -t odoo
+	docker run -p 127.0.0.1:8071:8069 --name odoo3 --link db:db -t odoo
+
+Please note that for plain use of mails and reports functionalities, when the 
+host and container ports differ (e.g. 8070 and 8069), one has to set, 
+in Odoo, Settings->Parameters->System Parameters (requires technical features), 
+web.base.url to the container port (e.g. 127.0.0.1:8069).
+
+## Access an existing database with a new Odoo instance
```

Comment:
I think this should be the "how to upgrade" section.  Using `docker exec` to upgrade a running container in-place is not something we recommend anywhere, especially since you then miss out on updates to the base image.  We instead canonically recommend that users download the updated image and replace their existing containers, as described here.